### PR TITLE
Better testimonials

### PR DESCRIPTION
**Pull request by ezhukov**
Enable testimonials and add two. Implement navigation on testimonials.

### CODE REVIEW

File: data/en/testimonial.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   bg_image_webp : "images/backgrounds/bg-testimonial.webp"
   testimonial_item :
     # testimonial item loop
-    - name : John Doe
-      image_webp : images/clients/client1.webp
-      date : Dec 26, 2023
+    - name : Arttu Heikkilä
+      image_webp : images/clients/arttu.webp
+      title : VP of Engineering at Sympa
       content : Lorem ipsum dolor sit amet, consectetur adipisicing elit. Praesentium minima ex, quia error, deserunt voluptatibus repellendus? Incidunt eligendi aliquam repellat molestias, vel sed optio rem odit quis ratione non consequuntur!
```
Comment (btbroot) [inline data/en/testimonial.yml]: This placeholder doesn't make sense to be taken (and published). Shouldn't we wait up for the actual testimonial instead?

File: data/en/testimonial.yml [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
       
     # testimonial item loop
-    - name : Jane Smith
-      image_webp : images/clients/client2.webp
-      date : Dec 26, 2023
-      content : Lorem ipsum dolor sit amet, consectetur adipisicing elit. Praesentium minima ex, quia error, deserunt voluptatibus repellendus? Incidunt eligendi aliquam repellat molestias, vel sed optio rem odit quis ratione non consequuntur!
+    - name : Soma Muthumanickam
+      image_webp : images/clients/soma.webp
+      title : VP of Products & Engineering at Basemark
+      content : Al was doing a great job leading the Dev-Ops efforts at Basemark. He is very strong technically and we could always rely on him to get things done. I particularly appreciated his knowledge of modern BDD approaches.
```
Comment (btbroot) [inline data/en/testimonial.yml]:
```suggestion
      content : Al was doing a great job leading the DevOps efforts at Basemark. He is very strong technology-wise, and we could always rely on him to get things done. I particularly appreciated his knowledge of modern Behavior Driven Development approaches.
```

File: themes/meghna-hugo/layouts/partials/testimonial.html
Comment (btbroot) [inline themes/meghna-hugo/layouts/partials/testimonial.html:1]: Add their LinkedIn profile links as well, maybe?

**Review by btbroot — CHANGES_REQUESTED** [review]
Largely fine, with few remarks to be addressed

File: data/en/testimonial.yml (hunk 1 above)
Comment (ezhukov) [reply data/en/testimonial.yml]: Yes, absolutely, that's why it's still in draft.

**Review by btbroot — APPROVED** [review]
Fine

**Pull request merged**